src/recursion.js

```diff
@@ -71,7 +71,7 @@ var multiply = function(x, y) {
 };
 
 // 13. Write a function that divides two numbers without using the / operator or
-// Math methods.
+// Math methods to arrive at an approximate quotient (ignore decimal endings).
```

Comment:
This change is already merged into `master`. I'm a little confused why it's appearing in this PR. Have you rebased with `master`?

Reply:
Since I didn't realize that was even a thing, I'm going to go with "no".

Will do going forward.